content/src/main/content/jcr_root/apps/acs-tools/components/tag-maker/clientlibs/js/app.js

```diff
@@ -68,6 +68,9 @@ angular.module('acs-tools-tag-maker-app', ['ngFileUpload', 'acsCoral', 'ACS.Tool
                 tagIds: []
             };
 
+            NotificationsService.add('info',
```

Comment:
Can we use `NotificationsService.running(true);`  and then in `success(...) and or error(..)` and `NotificationsService.running(false);` 

https://github.com/Adobe-Consulting-Services/acs-aem-tools/blob/master/content/src/main/content/jcr_root/apps/acs-tools/components/base-page/clientlibs/js/angularjs/notifications.js